os/Kconfig

```diff
@@ -439,6 +439,10 @@ menu "Hardware Configuration"
 source arch/Kconfig
 endmenu
 
+menu "Board Selection"
```

Comment:
arch/Kconfig's title is "Hardware Selection". Let's change it as "Chip Selection".

Reply:
Ok

Reply:
And then let's remove arch/Kconfig and rename Kconfig.chip to Kconfig.

Reply:
Ok